src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -842,7 +842,7 @@ protected void AddBlockedVersions(List<NuGetVersion> blockedVersions)
             // add all the versions blocked to disable the update button
             foreach (var version in blockedVersions)
             {
-                _versions.Add(new DisplayVersion(version, string.Empty, isValidVersion: false));
+                _versions.Add(new DisplayVersion(version, additionalInfo: null, isValidVersion: false));
```

Comment:
I'm doing this to keep the same format like in https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs#L218